design/object-bucket/object-bucket.md

```diff
@@ -2,38 +2,46 @@
+  
+  Rook-Ceph will be the first operator to utilize these new APIs.  The APIs themselves will be temporarily implemented in the Rook project as a discrete package and later be submitted for acceptance into a repository with broader Kubernetes support (for instance, [Kubernetes Enhancements](https://github.com/kubernetes/enhancements)).  By implementing dynamic bucket creation in Rook-Ceph first, the object bucket APIs will be able to present a stronger case for adoption when submitted to the to-be-determined project.
 
-## Users
+## Glossary
 
-- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use existing Rook-Ceph CRDs to create Ceph Object Stores and StorageClasses to make them accessible to users.  
-- User: A Kubernetes cluster user with limited permissions, typically confined to CRUD operations on Kubernetes objects within a single namespace.  The user will request Ceph Object buckets by instantiating CephObjectBucketClaims.
+- Admin: A Kubernetes cluster administrator with RBAC permissions to instantiate and control access to cluster storage. The admin will use existing Rook-Ceph CRDs to create Ceph Object Stores as well as StorageClasses to make them accessible to users.  
+- User: A Kubernetes cluster user with limited permissions, typically confined to CRUD operations on Kubernetes objects within a single namespace.  The user will request Ceph Object buckets by instantiating ObjectBucketClaims.
+- ObjectBucketClaim: a namespaced, user facing Custom Resource Definition representing a request for a dynamically created bucket (similar in use to PersistentVolumeClaims).  ObjectBucketClaims will reference a StorageClass whose _provisioner:_ field contains the name of an object store provisioner capable of dynamically provisioning buckets.  ObjectBucketClaims serve to abstract the underlying object store provider from the user.
+- ObjectBucket: a non-namespace, admin facing Custom Resource Definition representing the fulfillment of a user request for a dynamically created bucket in an existing object store (similar in use to PersistentVolumes).  ObjectBuckets serve to provide discrete information about the bucket and object store provider to admins.
 
 ## Goals
 
-- Provide Rook-Ceph users the ability to dynamically provision object buckets via the Kubernetes API with a CephObjectBucketClaim CRD.
-- Enable cluster admin control over bucket creation and object store access via the Kubernetes API.
+- Provide Rook-Ceph users the ability to dynamically provision object buckets via the Kubernetes API
+- Enable cluster admin control over bucket creation and object store access via the Kubernetes API
 - Restrict users from creating buckets with their access keys outside the view of Kubernetes (e.g. via `s3cmd` CLI tool)
```

Comment:
_Restrict users from creating buckets with their access keys outside the view of Kubernetes (e.g. via `s3cmd` CLI tool)_
nit: "Restrict users using the acess key-pairs, generated by new bucket creation, from working outside the view of Kubernetes (e.g. via `s3cmd` CLI tool)"